scripts/upload_generic_dataset.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+
+import requests
+
+from hari_client import Config
+from hari_client import hari_uploader
+from hari_client import HARIClient
+from hari_client import models
+from hari_client.utils.upload import check_and_create_dataset
+from hari_client.utils.upload import check_and_upload_dataset
+
+if __name__ == "__main__":
+    # Argument parser setup.
+    parser = argparse.ArgumentParser(
+        description="Example script how to upload one image as a dataset to HARI"
+    )
+
+    # Add command-line arguments.
+
+    parser.add_argument(
+        "--dataset_name",
+        type=str,
+        help="Name of the dataset which should be uploaded",
+        required=True,
+    )
+
+    parser.add_argument(
+        "--image_url",
+        type=str,
+        help="Url of an image which should be uploaded as a dataset",
+        required=True,
+    )
+
+    # Parse the arguments.
+    args = parser.parse_args()
+
+    # Extract arguments.
+    dataset_name = args.dataset_name
+    image_url = args.image_url
+
+    # load hari client
+    config: Config = Config(_env_file=".env")
+    hari: HARIClient = HARIClient(config=config)
+
+    # create dataset
+    dataset_id = check_and_create_dataset(
+        hari=hari, dataset_name=dataset_name, user_group=None, is_anonymized=True
+    )
+    # TODO what is a good default user group, how should a customer select it
```

Comment:
this is how we do it in the quickstart, and in the docs we explain how to find out which one the user needs

```suggestion
    dataset_id = check_and_create_dataset(
        hari=hari, dataset_name=dataset_name, user_group="CHANGEME", is_anonymized=True
    )
    # TODO what is a good default user group, how should a customer select it
```